packages/client/src/views/glsp-projection-view.tsx

```diff
@@ -82,11 +79,7 @@ export class GLSPProjectionView extends ProjectedViewportView {
             [GridProperty.GRID_BACKGROUND_Y]: bounds.y + 'px',
             [GridProperty.GRID_BACKGROUND_WIDTH]: bounds.width + 'px',
             [GridProperty.GRID_BACKGROUND_HEIGHT]: bounds.height + 'px',
-            [GridProperty.GRID_BACKGROUND_ZOOM]: viewport.zoom + '',
-            [GridProperty.GRID_BACKGROUND_IMAGE]: this.getBackgroundImage(viewport, context, this.gridManager),
-            backgroundPosition: `${bounds.x}px ${bounds.y}px`,
-            backgroundSize: `${bounds.width}px ${bounds.height}px`
-            // we do not set the background-image directly in the style object, because we want to toggle it on and off via CSS
+            [GridProperty.GRID_BACKGROUND_ZOOM]: viewport.zoom + ''
```

Comment:
Question: Do you think it would be worth setting the variables in any case so they could be re-used even if the grid was not actually visible? I'm just wondering about that. We probably do not need it for the hidden rendering but for a deactivated grid someone might still use something like the zoom.

Reply:
Fine with me. I pushed an update